apps/user-portal/src/components/account-recovery-email.tsx

```diff
@@ -0,0 +1,271 @@
+                    });
+                    getProfileInfo()
+                        .then((res) => {
+                            setEmailAddr(res);
+                        });
+                    setIsEdit(false);
+                } else {
+                    onNotificationFired({
+                        description: t(
+                            "views:userProfile.notification.getProfileInfo.error.description"
+                        ),
+                        message: t(
+                            "views:userProfile.notification.getProfileInfo.error.message"
+                        ),
+                        otherProps: {
+                            negative: true
+                        },
+                        visible: true
+                    });
+                }
+            });
+    };
+
+    /**
+     * The following function handles the change of state of the input fields.
+     * The id of the event target will be used to set the state.
+     * @param event
+     */
+    const handleFieldChange = (event) => {
+        setEditedEmail(event.target.value);
+        event.preventDefault();
+    };
+
+    /**
+     * This function gets the email address from the response passed as the argument 
+     * and assigns it to email and editedEmail. 
+     * @param resp 
+     */
+    const setEmailAddr = (resp) => {
+        let emailAddress = "";
+        resp.emails
+            ? resp.emails.map((email) => {
+                emailAddress = email;
+            })
+            : null;
+        setEmail(emailAddress);
+        setEditedEmail(emailAddress);
+    };
+
+    /**
+     * This is called when the edit icon is clicked. 
+     * 
+     */
+    const handleEdit = () => {
+        setIsEdit(true);
+    };
+
+    /**
+     * This is called when the cancel button is pressed
+     */
+    const handleCancel = () => {
+        setIsEdit(false);
+    };
+
+    /**
+     * This function masks the email address passed as the argument and returns 
+     * the masked email address.  
+     * The text between the second character of the email and the @ sign is masked.
+     * @param {string} email 
+     */
+    const maskEmail = (email: string) => {
+        let mask = "";
+        let indexOfAt = email.indexOf("@");
+        let textToBeMasked = email.slice(2, indexOfAt);
+
+        for (let i = 0; i < textToBeMasked.length; i++) {
+            mask += "*";
+        }
+
+        let maskedEmail = email.replace(textToBeMasked, mask);
+
+        return maskedEmail;
+    }
+
+    /**
+     * This function returns the EditSection component and the recovery option 
+     * elements based on if the edit icon has been clicked
+     */
+    const showEditView = () => {
+        if (!isEdit) {
+            return (<Grid padded>
+                <Grid.Row columns={2}>
+                    <Grid.Column width={11} className="first-column">
+                        <List.Content floated="left">
+                            <ThemeIcon
```

Comment:
Can we align this Icon with the list title?

Reply:
The icon is now aligned correctly. 
![image](https://user-images.githubusercontent.com/20745428/66900547-a5069a00-f01a-11e9-9c6e-f12245cddff0.png)